app/views/products/show.html.erb

```diff
@@ -67,10 +69,21 @@
       </tbody>
     </table>
   </div>
+  <!-- <% if @cart %>
+    <% @cart.line_items.each do |line_item| %>
+      <%= line_item.product.name %> - <%= line_item.quantity %>
+    <% end %>
+
+  <% else %>
+    <p>Your cart is empty.</p>
+  <% end %> -->
```

Comment:
if this code is a comment, please, remove it

Reply:
thanks, we forgot to remove it